_source/_posts/2022-06-02-java-rest-api-helidon.adoc

```diff
@@ -0,0 +1,339 @@
+---
+layout: blog_post
+title: "Create a Java REST API with Helidon"
```

Comment:
"Fire up a Java REST API with Helidon" might be good too. What do you think @havi?